migrations/20200604111227_create_commit_diagnosis_table.ts

```diff
@@ -0,0 +1,17 @@
+import * as Knex from "knex";
+
+const tableName = "commit_diagnosis";
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.createTable(tableName, (builder: Knex.TableBuilder) => {
+    builder.increments("id").primary();
+    builder.text("commit_id").notNullable();
+    builder.text("rule").notNullable();
+    builder.jsonb("data");
```

Comment:
For convenience, we can indicate no data with empty object `{}` and set it non-nullable.